cotrollers/articles.controller.js

```diff
@@ -0,0 +1,9 @@
+const { getArticleById } = require("../models/articles.model")
+
+exports.getArticleById = (req, res, next) => {
+    const {article_id} = req.params
+    getArticleById(article_id).then(article => {
+        console.log(article[0])
+        res.status(200).send({article: article[0]})

```

Comment:
Ideally your model should send back a single article, not an array. While this works and there is nothing functionally wrong here, it could do with tidying up.